_data/ome2021.json

```diff
@@ -584,6 +586,7 @@
             {
               "@type": "Event",
               "name": "IDR Workshop",
+              "description": "This session will first explore metadata associated to IDR data using the Web Interface. Then, we will demonstrate how to retrieve data and metadata using the APIs.",
```

Comment:
Introduction to IDR ecosystem will be shown on concrete examples of IDR studies. Images and metadata mining and comparison workflows will be presented. This workshop is going to show both to those who want to script as well as to those who do not how to get value from IDR.  The first, hands-on section will show the cross-studies searches using OMERO.web. The second section will expand on these possiblities showing complex metadata mining and plotting as well as image reanalysis in Jupyter notebooks.

Reply:
Trying to address @sbesson 's encouragement for longer description.